src/Elmish.WPF/Binding.fs

```diff
@@ -432,6 +403,31 @@ type Binding private () =
     } |> createBinding
 
 
+  /// <summary>
+  ///   Creates a one-way binding to a sequence of items, each uniquely
+  ///   identified by the value returned by <paramref name="getId"/>. The
+  ///   binding is backed by a persistent <c>ObservableCollection</c>, so only
+  ///   changed items (as determined by <paramref name="itemEquals" />) will be
+  ///   replaced. If the items are complex and you want them updated instead of
+  ///   replaced, consider using <see cref="subModelSeq" />. This method is
+  ///   implemented by calling <see cref="oneWaySeqLazy" /> with <c>refEq</c>
+  ///   for <c>equals</c> and <c>id</c> for <c>map</c>.
+  /// </summary>
+  /// <param name="get">Gets the collection from the model.</param>
+  /// <param name="itemEquals">
+  ///   Indicates whether two collection items are equal. Good candidates are
+  ///   <c>elmEq</c>, <c>refEq</c>, or simply <c>(=)</c>.
+  /// </param>
+  /// <param name="getId">Gets a unique identifier for a collection
+  /// item.</param>
+  static member oneWaySeq
+      (get: 'model -> #seq<'a>,
+       itemEquals: 'a -> 'a -> bool,
+       getId: 'a -> 'id)
+      : string -> Binding<'model, 'msg> =
+    Binding.oneWaySeqLazy(get, refEq, id, itemEquals, getId)
```

Comment:
> As I understand your comment, your preference is to have `Binding.oneWaySeq` call `Binding.oneWaySeqLazy`. That is my preference, too.

Yep.  In particular by defining `Binding.oneWaySeqLazy` before `Binding.oneWaySeq` like this.

I will leave this for you to resolve to ensure that we are on the same page here.